Source/Include/swigwarn.h

```diff
@@ -328,6 +328,15 @@
 
 /* please leave 890-899 free for Go */
 
-/* -- User defined warnings (900 - 999) -- */
+#define WARN_OBJC_TYPEMAP_OBJCTYPE_UNDEF     900
+#define WARN_OBJC_TYPEMAP_IMTYPE_UNDEF       901
+#define WARN_OBJC_TYPEMAP_OBJCIN_UNDEF       902
+#define WARN_OBJC_TYPEMAP_OBJCOUT_UNDEF      903
+#define WARN_OBJC_MULTIPLE_INHERITANCE       904
+#define WARN_OBJC_INTERFACE_MOD              905
+#define WARN_OBJC_IMPLEMENTATION_MOD         906
 
+/* please leave 900-919 free for ObjectiveC */
+
+/* -- User defined warnings (920 - 999) -- */
```

Comment:
900-999 are documented in the manual as reserved for use by the user (see `Warnings.html`), so you can't just commandeer them like this.

There are unused ranges in the block marked for use by target language modules - e.g. 760-799.